### DISCUSSION TITLE
show action when i have permission

### DISCUSSION THREAD

**Discussion by xx1196** [rappasoft/laravel-livewire-tables#1803] [Q&A] [answered]
if i have thos button group:ButtonGroupColumn::make('Actions')
                ->attributes(function ($row) {
                    return [
                        'class' => 'space-x-2',
                    ];
                })
                ->buttons([
                    LinkColumn::make('Edit')
                        ->title(fn($row) => 'Edit ' . $row->name)
                        ->location(fn($row) => route('users.edit', $row))
                        ->attributes(function ($row) {
                            return [
                                'target' => '_blank',
                                'class' => 'underline text-blue-500 hover:no-underline',
                            ];
                        }),
                ]),:how y can show or hide if i have the permission like users.edit in laravel spatie permissions?

**Comment by lrljoe** [comment, marked as answer]
I believe there's still a hideIf() method, which should be in the docs.  If it's not working, or the docs aren't clear, give me a tag
